docs/source/tutorials/object_detection.ipynb

```diff
@@ -513,7 +633,7 @@
     "assert 50 in issue_idx and issue_idx[0] == 50\n",
     "assert 16 in issue_idx and issue_idx[1] == 16\n",
     "assert 31 in issue_idx and issue_idx[2] == 31\n",
-    "assert 62 in issue_idx and issue_idx[6] == 62 \n",
+    "assert 62 in issue_idx and issue_idx[9] == 62 \n",
```

Comment:
did you verify changing 6 to 9 here is ok?
Please confirm you understand why this value was 6 before?